ui/src/tool-bar/tool-bar.tsx

```diff
@@ -376,8 +376,7 @@ export const ToolBar: React.FC<ToolbarParams> = ({
                 <LocationTrackEditDialogContainer
                     onClose={() => setShowAddLocationTrackDialog(false)}
                     onSave={handleLocationTrackSave}
-                    locationTrackChangeTime={changeTimes.layoutLocationTrack}
-                    switchChangeTime={changeTimes.layoutSwitch}
+                    changeTimes={changeTimes}
```

Comment:
Olikos meillä muuten jatkoidiksenä, että container olisi vastuussa tarvittavien oheisdepsujen hakemisesta, eli changeTimesiä ei täältä (ja muista containereiden parenteista) passattaisi vaikka onkin näin tietysti mahdollista? (Helpottaisi containerin+sen lapsikomponentin uudelleenkäyttöä tarvittaessa)

Reply:
Enpäs miettinytkään tuota taas tätä tehdessä vaikka itse noita containereita refakkasinkin ihan hetki sitten, mutta olihan tuo kyllä ideana. Kyllähän nuo `changeTimes`:it voisi tosiaan käydä hakemassa vasta containereissa